aten/src/ATen/native/Convolution.cpp

```diff
@@ -301,6 +301,10 @@ at::Tensor conv1d(
 at::Tensor conv2d(
     const Tensor& input, const Tensor& weight, const Tensor& bias,
     IntArrayRef stride, IntArrayRef padding, IntArrayRef dilation, int64_t groups) {
+  if (input.type_id() == MkldnnCPUTensorId()) {
+    return at::mkldnn_convolution(
```

Comment:
just re-registering from the old PR that this should be folded into the existing dispatch for this op.